packages/@aws-cdk/aws-cloud9-alpha/test/integ.owner.ts

```diff
@@ -0,0 +1,35 @@
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import { Stack, App, StackProps, CfnOutput } from 'aws-cdk-lib';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { Construct } from 'constructs';
+import * as cloud9 from '../lib';
+import { ImageId, Owner } from '../lib';
+
+class Cloud9Env extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const vpc = new ec2.Vpc(this, 'VPC', {
+      restrictDefaultSecurityGroup: false,
+      maxAzs: 2,
+      natGateways: 1,
+    });
+
+    // create a cloud9 ec2 environment in a new VPC
+    const c9env = new cloud9.Ec2Environment(this, 'C9Env', {
+      vpc,
+      imageId: ImageId.AMAZON_LINUX_2,
+      owner: Owner.assumedRole('807336301584', 'Admin/test'),
```

Comment:
```suggestion
      imageId: cloud9.ImageId.AMAZON_LINUX_2,
      owner: cloud9.Owner.assumedRole('807336301584', 'Admin/test'),
```
After removing the redundant `import` statement.